style.css

```diff
@@ -750,3 +756,370 @@ ul {
+    margin-top: -8.5rem;
+  }
+
+  .buttons {
+    margin-left: -5.4rem;
+  }
+
+  /* About section */
+
+  .about-me {
+    display: grid;
+    flex-direction: row;
+    width: 100%;
+    height: 90vh;
+    padding: 56px 14px 0 16px;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-left: -2rem;
+  }
+
+  .about-text {
+    display: flex;
+    white-space: nowrap;
+    margin-left: 10%;
+    width: 100%;
+    margin-top: -2.9rem;
+  }
+
+  .summary {
+    margin-left: 10%;
+    width: 27.2rem;
+    height: 102px;
+    margin-top: -2rem;
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+  }
+
+  .connection2 {
+    margin-left: 10%;
+    margin-top: 0.5rem;
+  }
+
+  .social2 {
+    margin-left: 7%;
+  }
+
+  .buttons2 {
+    margin-left: 10%;
+  }
+
+  .skill-list {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    float: right;
+    margin-left: 35rem;
+    margin-top: -45rem;
+  }
+
+  .down-arrow {
+    margin-left: 40rem;
+  }
+
+  .languages-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages-list2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 3rem;
+  }
+
+  .javascript2 {
+    display: flex;
+    flex-direction: column;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .js-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .js-text {
+    align-self: center;
+    margin-top: 5.6rem;
+    margin-left: -1.5rem;
+  }
+
+  .html2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: -0.1rem;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .html-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .html-text {
+    align-self: center;
+    margin-top: 5.8rem;
+    margin-left: -2.69rem;
+  }
+
+  .css2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0.021rem;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .css-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .css-text {
+    align-self: center;
+    margin-top: 5.5rem;
+    margin-left: -3.4rem;
+  }
+
+  .frameworks-text {
+    margin-left: 46.2rem;
+    margin-top: 6.2rem;
+  }
+
+  .right-arrow {
+    margin-left: -2rem;
+    margin-top: 6.4rem;
+  }
+
+  .right-arrow2 {
+    margin-left: -5.2rem;
+    margin-top: 11rem;
+  }
+
+  .separator-bottom {
+    margin-left: -32rem;
+    margin-top: -1px;
+  }
+
+  .skills-text {
+    margin-left: 46.2rem;
+    margin-top: 9.7rem;
+  }
+
+  .separarot-bottom2 {
+    margin-left: -35.5rem;
+    margin-top: -1rem;
+  }
+
+  /* contact section */
+
+  .contact-form {
+    width: 100%;
+    background-size: cover;
+    background-image: url(media/contact-form-background.svg);
+  }
+
+  .contact-container {
+    width: 100%;
+  }
+
+  .contact-text {
+    width: 35%;
+    height: 102px;
+    top: 68px;
+    margin-left: 380px;
+  }
```

Comment:
- [ ] Please make sure the text in the "contact-me" section is aligned to the center as specified in the [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=YF5TFeKU8vzMU6sB-0)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230618525-81547257-fef9-4081-8994-a194ce4fe2d6.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/230618701-c665a8cf-2916-4920-8988-8d2ac78a9749.png)